Analysis/Tasks/PWGHF/qaTaskSimple.cxx

```diff
@@ -234,47 +266,55 @@ struct QATrackingResolution {
     o2fw::AxisSpec impactParZErrorAxis{nBinsImpactParameter, impactParameterResolutionRange[0],
                                        impactParameterResolutionRange[1]};
 
+    TString commonTitle = "";
+    if (pdgCode.value != 0) {
+      commonTitle += Form("PDG %i", pdgCode.value);
+    }
+    if (useOnlyPhysicsPrimary.value == 1) {
+      commonTitle += " Primary";
+    }
+
     // Eta
-    histos.add("eta/etaDiffMCReco", ";#eta_{MC} - #eta_{Rec}", o2fw::kTH1D, {deltaEtaAxis});
-    histos.add("eta/etaDiffMCRecoVsEtaMC", ";#eta_{MC} - #eta_{Rec};#eta_{MC}", o2fw::kTH2D, {deltaEtaAxis, etaAxis});
-    histos.add("eta/etaDiffMCRecoVsEtaReco", ";#eta_{MC} - #eta_{Rec};#eta_{Rec}", o2fw::kTH2D, {deltaEtaAxis, etaAxis});
+    histos.add("eta/etaDiffMCReco", commonTitle + ";#eta_{MC} - #eta_{Rec}", o2fw::kTH1D, {deltaEtaAxis});
+    histos.add("eta/etaDiffMCRecoVsEtaMC", commonTitle + ";#eta_{MC} - #eta_{Rec};#eta_{MC}", o2fw::kTH2D, {deltaEtaAxis, etaAxis});
+    histos.add("eta/etaDiffMCRecoVsEtaReco", commonTitle + ";#eta_{MC} - #eta_{Rec};#eta_{Rec}", o2fw::kTH2D, {deltaEtaAxis, etaAxis});
```

Comment:
I think `_{MC}` is misleading. Reconstruction and generator levels are both MC. I would change it to `_{Gen}`.
Also, resolution is usually defined the other way around, i.e. as `rec - gen`, since `gen` is the reference and resolution expresses the deviation of `rec` from it.

Reply:
Definition not clarified.